tools/interactive/interactivetool_climate_notebook.xml

```diff
@@ -23,7 +23,8 @@
 
         #if $input:
             #set $cleaned_name = re.sub('[^\w\-\.]', '_', str($input.element_identifier))
```

Comment:
I can not edit this file, seems I don't have permissions. This is a good idea I think. To make it more stable we take the ID of the dataset.

get -t name -i '${input.hid}' &&
ln -sf '/import/${input.hid}' './jupyter/data/${cleaned_name}' &&

Reply:
I am not sure why you can't edit it. I updated it and also gave you access to the repo.